package/MDAnalysis/core/groups.py

```diff
@@ -2676,12 +2676,12 @@ def unique(self):
            >>> ag
            <AtomGroup with 6 atoms>
            >>> ag.ix
-           array([2, 1, 2, 2, 1, 0])
+           array([2, 1, 2, 2, 1, 0], dtype=int64)
```

Comment:
When I return a numpy array in the terminal, the dtype usually doesn't appear, can you explain why you added the dtype?

Reply:
After adding the imports, the output.txt from `make doctest` showed:
```
File "..\..\..\MDAnalysis\core\groups.py", line ?, in default
Failed example:
    ag2.ix
Expected:
    array([2, 1, 0])
Got:
    array([2, 1, 0], dtype=int64)
```

When I ran the commands in my terminal (Git Bash, Windows 10, Python 3.11.0, miniconda), I get a similar result:
```
>>> ag = u.atoms[[2, 1, 0]]
>>> ag2 = ag.asunique(sorted=False)
>>> ag2 is ag
True
>>> ag2.ix
array([2, 1, 0], dtype=int64)
```

So I thought it made sense to add the dtype to resolve those failing tests. Do you think it might be because I am on Windows? I have been meaning to switch to Linux but used Windows for this PR because I had the dev environment set up already.

Reply:
odd, not sure why my terminal is showing something different. If you need the dtype to pass the test then feel free to continue including it!

Reply:
Do you think it would be worthwhile to ask a third person to try these commands in their terminal to check whether it makes sense to add the dtype or not?

Reply:
@hmacdope are y'all trying to standardize this across the docs in one direction or the other?

Reply:
Whatever makes the doctest work. :)

Reply:
Sounds good! @orionarcher is there anything else I should add or edit in this PR? The tip about imports was quite helpful, and I will keep it in mind if I work on this issue again in the future.